memory-db/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
-parity-util-mem = { version = "0.9.0", default-features = false, features = ["hashbrown"] }
+parity-util-mem = { version = "0.10.0", default-features = false, features = ["hashbrown"] }
 hash-db = { path = "../hash-db", default-features = false, version = "0.15.2"}
 hashbrown = { version = "0.9.1", default-features = false, features = [ "ahash" ] }
```

Comment:
```suggestion
hashbrown = { version = "0.11", default-features = false, features = [ "ahash" ] }
```

Reply:
this is required because of `parity-util-mem` 0.10 impls it for the new version only